Resources/Prototypes/Objectives/thief.yml

```diff
@@ -205,15 +205,16 @@
     difficulty: 1
 
 - type: entity
-  parent: BaseThiefStealObjective
-  id: ClothingHeadsetAltMedicalStealObjective
+  parent: BaseThiefStealCollectionObjective
+  id: ClothingCommandHeadsetStealObjective #imp edit: i'm branching this out from just the CMO's headset to all command headsets so there is less of a blatant kick-me sign on the CMO
   components:
-  - type: NotJobRequirement
-    job: ChiefMedicalOfficer
   - type: StealCondition
-    stealGroup: ClothingHeadsetAltMedical
+    stealGroup: CommandHeadset
+    minCollectionSize: 4
+    maxCollectionSize: 12
+    verifyMapExistence: false
```

Comment:
this should not be false: there's no guarantee, especially on lowpop maps, that all of these exist. when it's set to true, it also clamps the upper value to match however many actually exist

Reply:
does verifyMapExistence count what characters are wearing at roundstart? i set this to false because i'm attempting to accomodate that the eligible headset economy is constantly fluctuating

Reply:
i'm unsure, but it's important to note that thief is the most common lowpop gamemode below the 5 player ready count and reach definitely doesn't have enough headsets for this objective: if you have a captain and hos, and only roll 4 headsets, then its technically possible, but any other situation is an impossible objective